[2025-09-01T04:22:53.945+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.archive manual__2025-09-01T04:22:43.695084+00:00 [queued]>
[2025-09-01T04:22:53.985+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.archive manual__2025-09-01T04:22:43.695084+00:00 [queued]>
[2025-09-01T04:22:54.003+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-01T04:22:54.076+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): archive> on 2025-09-01 04:22:43.695084+00:00
[2025-09-01T04:22:54.118+0000] {standard_task_runner.py:60} INFO - Started process 836 to run task
[2025-09-01T04:22:54.148+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'archive', 'manual__2025-09-01T04:22:43.695084+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmprpe5r22c']
[2025-09-01T04:22:54.159+0000] {standard_task_runner.py:88} INFO - Job 13: Subtask archive
[2025-09-01T04:22:54.219+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-01T04:22:54.345+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline.archive manual__2025-09-01T04:22:43.695084+00:00 [running]> on host c7e4bfbd9438
[2025-09-01T04:22:54.540+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='archive' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T04:22:43.695084+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-01T04:22:43.695084+00:00'
[2025-09-01T04:22:54.542+0000] {logging_mixin.py:188} INFO - Running task: ARCHIVE
[2025-09-01T04:22:54.543+0000] {logging_mixin.py:188} INFO - Simulating the archival of the original raw data...
[2025-09-01T04:22:54.543+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-01T04:22:54.596+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=archive, execution_date=20250901T042243, start_date=20250901T042253, end_date=20250901T042254
[2025-09-01T04:22:54.664+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-01T04:22:54.700+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
